x-pack/plugins/enterprise_search/server/index_management/setup_indices.ts

```diff
@@ -98,23 +171,46 @@ const indices: IndexDefinition[] = [
     aliases: ['.elastic-connectors-sync-jobs'],
     mappings: {
       _meta: {
-        version: '1',
+        version: 1,
       },
       properties: {
         completed_at: { type: 'date' },
-        connector: { properties: connectorMappingsProperties },
```

Comment:
removing the connector object, see: https://elastic.slack.com/archives/C01795T48LQ/p1666978938392449?thread_ts=1666967212.488069&cid=C01795T48LQ